gammapy/cube/psf_map.py

```diff
@@ -3,7 +3,7 @@
 import astropy.io.fits as fits
 import astropy.units as u
 from astropy.coordinates import Angle
-from gammapy.cube import PSFKernel
+from gammapy.cube.psf_kernel import PSFKernel
```

Comment:
Why can't you import from `gammapy.cube` directly?

Reply:
I don't understand why, but it was giving an import error.

Reply:
Solved in https://github.com/gammapy/gammapy/pull/2358/commits/de85dbd70a9936fa6335e2d3489b283d10027b3f